core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -656,16 +656,6 @@ public void addConfiguration(Configuration config) {
     super.addConfiguration(config);
   }
 
-  /**
-   * @deprecated since 1.9.2; will be removed in 2.0.0 to eliminate commons config leakage into
-   *             Accumulo API
-   */
-  @Deprecated
-  @Override
-  public void addConfiguration(Configuration config, boolean asInMemory) {
```

Comment:
Did this actually go out in 1.9.2?

Reply:
No. This was specifically added in your previous PR for 1.9.3 https://github.com/apache/accumulo/pull/659/files

Recall that we deprecated the superclass of ClientConfiguration, and added in all the methods inherited from the superclass, to mark them with the Deprecated annotation. This was a new method between 1.6 and 1.10, that was included here merely to declare it as Deprecated (since once cannot annotate the superclass directly, only its inherited methods).

Reply:
okay so it's just a docs error on the deprecated since line. lgtm.

Reply:
@busbey Not necessarily a docs error. There was a discussion about how to annotate the deprecated override on your original PR. This new method was added to the superclass by the dependency update during 1.9.3 development... but the superclass itself was deprecated in 1.9.2. Since these overridden methods are being used as a proxy to annotate that the superclass was deprecated, it could go either way.